### PR TITLE
fix node install

### PR DESCRIPTION
**Pull request by snopoke**
based on https://github.com/nodejs/docker-node/blob/master/5.12/Dockerfile
@millerdev

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,14 +3,14 @@ MAINTAINER Dimagi <devops@dimagi.com>
...
 
-RUN curl -sL https://deb.nodesource.com/setup_5.x | bash - \
- && apt-get -y install \
-    nodejs \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/* \
- && mkdir /vendor
+RUN mkdir /vendor
+
+RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.gz" \
```
Comment (millerdev) [inline Dockerfile]: Why separate this into two separate `RUN` commands? It's my understanding that each command creates a new layer, and it's desirable to reduce the number of layers.

Comment (snopoke) [inline Dockerfile]: It was a compromise between reducing layers and readability. I think it makes sense to group related commands into one but I'm less keen to have completely unrelated stuff together.

I 'feel' like each RUN command should do one task (install Java, install pip requirements, install nmp etc.)
